testdata/testinput1

```diff
@@ -6721,4 +6715,16 @@ $/x
 /65 00 64/hex
     e\0d
 
+/[[:digit:]-   ]/xx
+    1
+    -
+\= Expect no match
+    z
```

Comment:
better check for "space" (`\ \`) as not matching

Reply:
Great, done